test/compile.test.ts

```diff
@@ -84,26 +84,29 @@ for (const unitName of unitTestDirs) {
       format: (pkgJson.main ? 'cjs' : 'esm') as 'cjs' | 'esm',
     }
 
-    // build dist file and minified file
-    await bundle(inputFileName, { ...baseOptions, file: distFile })
-    await bundle(inputFileName, {
-      ...baseOptions,
-      file: minifiedDistFile,
-      minify: true,
-    })
-
+    // build dist file
+    const isTsFile = ['.ts', '.tsx'].includes(extname(inputFileName))
     const compareSnapshot: CompareFn = (
       bundledAssetContent,
       currentOutputSnapshot,
     ) => {
       expect(bundledAssetContent).toEqual(currentOutputSnapshot)
     }
 
+    await bundle(inputFileName, { ...baseOptions, file: distFile })
     await compareOrUpdateSnapshot(distFile, unitName, compareSnapshot)
-    await compareOrUpdateSnapshot(minifiedDistFile, unitName, compareSnapshot)
-
-    if (['.ts', '.tsx'].includes(extname(inputFileName))) {
+    if (isTsFile) {
       await compareOrUpdateSnapshot(distTypesFile, unitName, compareSnapshot)
+    }
+
+    // build minified file
+    await bundle(inputFileName, {
```

Comment:
It will clean the dist directory when executing the `bundle` function. So, if we keep this as before, I think we need to provide a new option in `BundleConfig`, or even support a new cli arg like `--clean` or `--noClean`, so that we can specify whether the dist needs to be deleted when the bundle is executed.

Reply:
Do you have any suggestions?

Reply:
Good call, we can clean by default, and add one more`--no-clean` arg for not doing it. Also need a test for that!

Reply:
Just add a `--no-clean` option and add test to integration tests.

Btw, I am a little confused, I can't figure out what difference between `cli.test` and `integration.test`. I feel like both are integration test cases.

Reply:
Maybe I could answer that for you, `cli` tests focus on targeting the CLI itself and the `--args` whereas `integration` is more of an E2E test that runs to meet the specific use cases.

Reply:
yeah would be good if we can move to cli test later, but it's not blocking for merging

Reply:
> Maybe I could answer that for you, `cli` tests focus on targeting the CLI itself and the `--args` whereas `integration` is more of an E2E test that runs to meet the specific use cases.

thanks!